docs/code_ref/coordinates/rotatedsunframe.rst

```diff
@@ -0,0 +1,390 @@
+   '2001-01-04 00:00:00.000' '2001-01-05 00:00:00.000'
+   '2001-01-06 00:00:00.000'], rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate for 'earth'>): (Tx, Ty, distance) in (arcsec, arcsec, AU)
+      [(-847.35767803, 419.05107437, 0.9822281 ),
+       (-764.81081635, 428.25724753, 0.9813374 ),
+       (-644.29157717, 437.09454986, 0.98056026),
+       (-491.84018851, 445.01082595, 0.97993388),
+       (-315.11434361, 451.4724754 , 0.97948809),
+       (-123.        , 456.        , 0.97924388),
+       (  74.8471119 , 458.20167789, 0.97921235),
+       ( 268.50338021, 457.80291945, 0.97939415),
+       ( 448.29323287, 454.66911128, 0.97977955),
+       ( 605.28861971, 448.82020568, 0.98034895),
+       ( 731.76302454, 440.43591752, 0.98107395)]>
+
+Transforming into RotatedSun frames
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+So far, all of the examples show transformations with the ``RotatedSun*`` frame as the starting frame.
+The ``RotatedSun*`` frame can also be the destination frame, which can be more intuitive in some situations and even necessary in some others (due to API limitations).
+Let's use a coordinate from earlier, which represents the coordinate in a "real" coordinate frame::
+
+  >>> coord = rs_hgc.transform_to(rs_hgc.base)
+  >>> coord
+  <HeliographicCarrington Coordinate (obstime=2020-03-04T00:00:00.000): (lon, lat, radius) in (deg, deg, km)
+      (45.13354448, 20., 695700.)>
+
+If we create a ``RotatedSun*`` frame for a different base time, we can represent that same point using coordinates prior to differential rotation::
+
+  >>> rs_frame = RotatedSunFrame(base=f.HeliographicCarrington(obstime=coord.obstime), rotated_time="2020-03-06 12:00")
+  >>> rs_frame
+  <RotatedSunHeliographicCarrington Frame (base=<HeliographicCarrington Frame (obstime=2020-03-04T00:00:00.000)>, duration=2.5 d, rotation_model=howard)>
+
+  >>> new_coord = coord.transform_to(rs_frame)
+  >>> new_coord
+  <RotatedSunHeliographicCarrington Coordinate (base=<HeliographicCarrington Frame (obstime=2020-03-04T00:00:00.000)>, duration=2.5 d, rotation_model=howard): (lon, lat, radius) in (deg, deg, km)
+      (10., 20., 695700.)>
+
+There coordinates are stored in the ``RotatedSun*`` frame, but it can be useful to "pop off" this extra layer and retain only the coordinate representation in the base coordinate frame.
+There is a convenience method called `~sunpy.coordinates.metaframes.RotatedSunFrame.as_base()` to do exactly that.
+Be aware the resulting coordinate does *not* point to the same location in inertial space, despite the superficial similarity.
+Essentially, the component values have been copied from one coordinate frame to a different coordinate frame, and thus this is not merely a transformation between coordinate frames::
+
+  >>> new_coord.as_base()
+  <HeliographicCarrington Coordinate (obstime=2020-03-04T00:00:00.000): (lon, lat, radius) in (deg, deg, km)
+      (10., 20., 695700.)>
+
+Advanced uses of RotatedSunFrame
```

Comment:
I think these examples would be better in individual gallery examples, since this is already quite an individually long bit of documentation.